CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# [v0.8.0](https://github.com/hyperium/tonic/compare/v0.7.2...v0.8.0) (Unreleased)
+
+### Breaking Changes
+
+* **build:** Remove `compression` feature
+* **build:** Generated `accept_gzip` and `send_gzip` replaced with `accept_compressed` and `send_compressed`
+* **tonic:** Remove `compression` feature, add `gzip`
+* **tonic:** `accept_gzip` and `send_gzip` replaced with `accept_compressed` and `send_compressed`
+
+### Features
+
+* Code generated with `tonic-build` will now work regardless of whether `tonic` compression support is enabled
```

Comment:
I believe we usually fills in the changelog when doing releases so perhaps this change should be removed. @LucioFranco knows :)

Reply:
Oh Ya good point if you back this out we will add this during release.

Reply:
Dropped that commit off the end.